src/index.html

```diff
@@ -7,8 +7,52 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com"><link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- <h1>Moyo header</h1> -->
+
+    <header class="header">
+      <a href="*">
+        <img src="images/logo.png"
+          alt="logo-moyo"
+          class="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="list-item">
```

Comment:
```suggestion
          <li class="nav_item">
```
it would be better to keep class names in a common structure